src/main/java/gold/milli/initialproject/mapper/UpdateUserRequestMapper.java

```diff
@@ -0,0 +1,11 @@
+package gold.milli.initialproject.mapper;
+
+import gold.milli.initialproject.entity.UpdateUserRequestDto;
+import gold.milli.initialproject.entity.User;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface UpdateUserRequestMapper {
```

Comment:
move this to UserMapper

Reply:
and also never map dto to entity
load it from database.

but map entity to dto is ok